resolwe_bio/tests/test_upload.py

```diff
@@ -14,7 +13,14 @@ def test_bam_upload(self):
         self.assertFiles(upload_bam, 'bai', 'alignment_bam_upload_index.bai')
 
         inputs = {"src": "alignment_position_sorted.bam", "src2": "alignment_bam_upload_index.bai"}
-        self.assertRaises(mongoengine.ValidationError, self.run_processor, "import:upload:mapping-bam-indexed", inputs)
+        try:
+            import mongoengine
+            self.assertRaises(
+                mongoengine.ValidationError, self.run_processor, "import:upload:mapping-bam-indexed", inputs)
```

Comment:
```from .utils import ValidationError```

that can either be a ```mongoengine.ValidationError``` or a ```django.core.exceptions.ValidationError``` or some other validation error that we will use for validating the postgres model.